modules/openxr/openxr_api.cpp

```diff
@@ -3095,11 +3094,21 @@ bool OpenXRAPI::is_environment_blend_mode_supported(XrEnvironmentBlendMode p_ble
 }
 
 bool OpenXRAPI::set_environment_blend_mode(XrEnvironmentBlendMode p_blend_mode) {
+	if (emulate_environment_blend_mode_alpha_blend && p_blend_mode == XR_ENVIRONMENT_BLEND_MODE_ALPHA_BLEND) {
```

Comment:
`get_supported_environment_blend_modes` needs to be altered too, so that if `XR_ENVIRONMENT_BLEND_MODE_ALPHA_BLEND` is not in the returned list, but `emulate_environment_blend_mode_alpha_blend` is true, it should be added. It will be `get_supported_environment_blend_modes` with which the user determines passthrough is available. (the `is_passthrough_supported` should be marked deprecated if you haven't already)